data/.gitignore

```diff
@@ -1 +1,2 @@
-/accountdatabase.json
\ No newline at end of file
+/accountdatabase.json
+/metadata.json
```

Comment:
Maybe we should create a dir for only these data files. Either a new dir or we move away what's currently there that's not part of it. What do you think?

Reply:
I don't mind. Happy to put them in a separate directory but I don't think it's terrible having them at the top level.